handlers/base_commands.py

```diff
@@ -25,38 +29,55 @@ async def commands(message: types.Message):
         content.t(language_code(message), 'help_message').format(VERSION),
         disable_notification=True,
         parse_mode='Markdown',
-        reply_markup=await kb.main(message.from_user.id)
+        reply_markup=await kb.main(message)
     )
 
 
-@dp.message_handler(regexp='⚙️ регистрация')
+@dp.message_handler(regexp = '⚙️ регистрация')
+@dp.message_handler(regexp = '⚙️ registration')
```

Comment:
тут наверно можно объединить два хендлера

```suggestion
@dp.message_handler(regexp = '⚙️ (registration|регистрация)')
```

Reply:
Не уверен, что так лучше. Может стоит эти регекспы собрать тоже в каком-нибудь модуле или в константе вверху модуля? И тогда тут будет что-то вроде

```suggestion
@dp.message_handler(regexp = REGISTRATION_HANDLER_REGEXP)
```